flow/layers/platform_view_layer.cc

```diff
@@ -31,10 +31,6 @@ void PlatformViewLayer::Preroll(PrerollContext* context,
                                            context->mutators_stack);
   context->view_embedder->PrerollCompositeEmbeddedView(view_id_,
                                                        std::move(params));
-#if defined(LEGACY_FUCHSIA_EMBEDDER)
```

Comment:
Removed this block, set_needs_system_composite is set by CheckForChildLayerBelow